sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -850,3 +851,29 @@ ALTER TABLE `form_vitals` ADD `inhaled_oxygen_concentration` float(4,1) DEFAULT
 #EndIf
 
 UPDATE `list_options` SET `notes` = 'LOINC:11502-2' WHERE `list_options`.`list_id` = 'Clinical_Note_Type' AND `list_options`.`option_id` = 'laboratory_report_narrative';
+
+#IfMissingColumn patient_data care_team_status
+ALTER TABLE patient_data ADD COLUMN care_team_status VARCHAR(100) NULL DEFAULT NULL;
+#EndIf
+
+#IfNotTable patient_careteam_history
+CREATE TABLE `patient_careteam_history` (
+    `id` BIGINT(20) NOT NULL AUTO_INCREMENT
+    , `uuid` BINARY(16) NULL
+    , `care_team_provider` TEXT NULL
+    , `care_team_facility` TEXT NULL
+    , `date` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
+    , `pid` BIGINT(20) NOT NULL
+    , PRIMARY KEY (`id`)
+    , UNIQUE `uuid` (`uuid`)
+) ENGINE = InnoDB;
+#EndIf
```

Comment:
This bothers me. I don't oppose it but, I'd like to see this as a multi use table. First, most users are likely never going to use.  Second, I better review if a requirement in CCDA.:) I didn't see it!

Reply:
So if you want to make it multi-use that's fine.  I just need to handle at least two care team records for a patient and this seemed like the simplest route to get there.

Reply:
providers/users/address/facility would all be linked tables from patient_history the same as for any patient history items.
I get it where you are panning for eventual relationships and this would be the way to go however while thinking about how to handle history of names I reflected back to a convo I had with Robert D. concerning maintaining lots of different histories.
So if not to much trouble could you rename the table patient_data_history or some such name?